docs/release-notes-v7.0.12.md

```diff
@@ -0,0 +1,29 @@
+# Percona Distribution for MongoDB 7.0.12 (2024-07-18)
+
+[Upgrade now](installation.md){.md-button}
+
+Percona Distribution for MongoDB is a freely available MongoDB database alternative, giving you a single solution that combines enterprise components from the open source community, designed and tested to work together. Its aim is to enable you to run and operate your
+MongoDB efficiently with the data being consistently backed up.
+
+Percona Distribution for MongoDB includes the following components:
+
+* *Percona Server for MongoDB* is a fully compatible source-available, drop-in replacement
+for MongoDB.
+
+* *Percona Backup for MongoDB* is a distributed, low-impact solution for achieving
+consistent backups of MongoDB sharded clusters and replica sets.
+
+This release of Percona Distribution for MongoDB is based on the production release of [Percona Server for MongoDB 7.0.12-7](https://docs.percona.com/percona-server-for-mongodb/7.0/release_notes/7.0.12-7.html) and [Percona Backup for MongoDB 2.5.0](https://docs.percona.com/percona-backup-mongodb/release-notes/2.5.0.html).
+
+## Release Highlights
+
+* Upgraded the MozJS/SpiderMonkey engine that MongoDB uses to execute server-side JavaScript code within the database server to the latest ESR (Extended Support Release)
```

Comment:
can we be specific to which version exactly?

Reply:
It's not stated by upstream:(